src/hooks/useDisputes.js

```diff
@@ -55,10 +55,35 @@ export default function useDisputes() {
 }
 
 export function useDispute(disputeId) {
-  const [disputeProcessed, setDisputeProcessed] = useState(null)
   const courtConfig = useCourtConfig()
   const now = useNow() // TODO: use court clock
+
   const { dispute, fetching } = useSingleDisputeSubscription(disputeId)
+  const disputeProcessed = useProcessedDispute(dispute)
+
+  const disputePhase = getPhaseAndTransition(dispute, courtConfig, now)
+  const disputePhaseKey = disputePhase
+    ? convertToString(Object.values(disputePhase)[0])
+    : ''
+
+  return useMemo(() => {
+    if (fetching || (dispute && !disputeProcessed)) {
+      return { fetching: true }
```

Comment:
Not sure how the rest of the project's hooks have been standardized on, but in this case, it might be nice to return an array instead, rather than two different type schemas.

So for example:

```
return [null, true] // fetching

return [{ ... }, false] // has information
```

Reply:
When the hook returns lots of different data we usually just use an object.

In this case i think your suggestion makes sense.

> But if all the other code is structured like how this hook is currently written, with an implicitly false fetching key when data exists

It's not, we can change it without extra effort